cli/integration/tests/waiter/test_cli.py

```diff
@@ -464,6 +464,7 @@ def test_ping_basic(self):
             self.assertIn('Successfully pinged', cli.stdout(cp))
             self.assertEqual(1, len(util.services_for_token(self.waiter_url, token_name)))
         finally:
+            util.kill_services_using_token(self.waiter_url, token_name)
```

Comment:
should we have a helper function in `util` that invokes boh these?

Reply:
Done, added a `kill_services` arg to `util.delete_token`